values.yaml

```
@@ -22,7 +22,7 @@ config:
 
 image:
   repository: "tryretool/backend"
-  tag: "latest"
+  tag: "2.66.65"
```

```suggestion
  tag: "X.Y.Z"
```

Discussed offline. Let's go with this.